metricbeat/module/openmetrics/collector/collector_test.go

```diff
@@ -22,11 +22,374 @@ package collector
 import (
 	"testing"
 
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/metricbeat/helper/openmetrics"
+	p "github.com/elastic/beats/v7/metricbeat/helper/prometheus"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/golang/protobuf/proto"
+	lbl "github.com/prometheus/prometheus/pkg/labels"
```

Comment:
Nit. why aliasing these packages? `prometheus` name is not used in other places. `labels` is a more generic name and used for some variables, but I would prefer to rename the variables, or aliasing to a more meaningful name as `prometheuslabels` instead of `lbl`
```suggestion
	"github.com/elastic/beats/v7/metricbeat/helper/prometheus"
	"github.com/elastic/beats/v7/metricbeat/mb"
	"github.com/golang/protobuf/proto"
	"github.com/prometheus/prometheus/pkg/labels"
```

Reply:
Changed references to `p` helper/prometheus, changed  `lbl` to `prometheuslabels`